### PR TITLE
react-native: fixed Animated.EventConfig

### PR DESCRIPTION
**Pull request by iyegoroff**
Docs - http://facebook.github.io/react-native/docs/animated#event

### CODE REVIEW

File: types/react-native/index.d.ts [hunk 1]
```diff
@@ -8465,8 +8465,8 @@ export namespace Animated {
     export function parallel(animations: Array<CompositeAnimation>, config?: ParallelConfig): CompositeAnimation;
 
     type Mapping = { [key: string]: Mapping } | AnimatedValue;
-    interface EventConfig {
-        listener?: ValueListenerCallback;
```
Comment (alloy) [inline types/react-native/index.d.ts]: Is this callback type unused now? If so, can you remove it?

**Review by alloy — COMMENTED** [review]
Looks good! Just one question after which this is good to merge imo.

File: types/react-native/index.d.ts (hunk 1 above)
Comment (iyegoroff) [reply types/react-native/index.d.ts]: No, it is used by Animated.Value

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7f0cd433de22a10d890c2e02a6a8cf23f7397617/types/react-native/index.d.ts#L8248-L8253

Comment (alloy) [reply types/react-native/index.d.ts]: Cool, thanks! (Sorry for the laziness, was on the road with my phone)

Comment (iyegoroff) [reply types/react-native/index.d.ts]: No problem, thanks!

**Pull request merged**